infinite_tracing/test/infinite_tracing/client_test.rb

```diff
@@ -66,10 +68,14 @@ def test_handles_server_disconnects
 
             total_spans = 5
             leftover_spans = 2
-            spans, segments = emulate_streaming_segments total_spans do |client, segments, server|
-              if segments.size == total_spans - leftover_spans
-                server.restart
-                client.restart
+
+            spans, segments = emulate_streaming_segments total_spans do |client, current_segments|
+              if current_segments.size == total_spans - leftover_spans
+                # we do this here instead of server_proc because we are testing that if the server restarts while the CLIENT is currently running (since we are not testing the server, just client behavior)
```

Comment:
I've been keeping code to 120 cols and comments to 80. Do we have an agent-wide set of rules we could add to RuboCop?

Reply:
I don't think we have a current agent wide rule, we could make a ticket to add that and see how many places we'll need to update according to rubocop. In the meantime though, I did break this up into a few more lines.